src/detection_win.cpp

```diff
@@ -7,6 +7,7 @@
 
 // Include Windows headers
 #include <windows.h>
+#include <Cfgmgr32.h>
```

Comment:
Why is this needed now?

Reply:
CM_DEVCAP_UNIQUEID is defined in that header file, so we have to include it.

Reply:
I don't see it listed here

 - https://msdn.microsoft.com/en-us/library/windows/hardware/mt813619%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396
 - https://github.com/msysgit/msysgit/blob/e6cc6bf1bba06c7029b2d59aedb0c975a0c4f6fe/mingw/include/ddk/cfgmgr32.h

But the docs seem to indicate it should be :+1:

> SPDRP_CAPABILITIES
> 
> The function retrieves a bitwise OR of the following CM_DEVCAP_Xxx flags in a DWORD. The device capabilities that are represented by these flags correspond to the device capabilities that are represented by the members of the DEVICE_CAPABILITIES structure. The CM_DEVCAP_Xxx constants are defined in Cfgmgr32.h.
>
> https://msdn.microsoft.com/en-us/library/windows/hardware/ff551967(v=vs.85).aspx